### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by intrudir**
Added 2 new resources
- Rev shell generator
- Trickest CVE repository

### CODE REVIEW

**Review by edoardottt — CHANGES_REQUESTED** [review]
Thank you so much @intrudir for the help!

I would like to use the official description of the services, in this case:

Trickest CVE: Gather and update all available and newest CVEs with their PoC.

Rev shell gen: Online Reverse Shell generator with Local Storage functionality, URI & Base64 Encoding, MSFVenom Generator, and Raw Mode. Great for CTFs.

**Comment by edoardottt** [comment]
Yeah you're right, it is obviously redundant... but for simplicity I prefer using the descriptions the services are using :)

**Comment by intrudir** [comment]
Sounds good to me. How do I approve your requested changes? I'm seeing "Only those with write access can merge"

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [ScanFactory](https://in.scanfactory.io/cvemon.html) - Realtime Security Monitoring
 - [Trend Micro Zero Day Initiative](https://www.zerodayinitiative.com/advisories/published/) - Publicly disclosed vulnerabilities discovered by Zero Day Initiative researchers
 - [Google Project Zero](https://bugs.chromium.org/p/project-zero/issues/list?can=1&q=&sort=-id&colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary) - Vulnerabilities including Zero Days
+- [Trickest CVE Repository](https://github.com/trickest/cve) - Almost every publicly available CVE PoC.
```
Comment (edoardottt) [inline README.md:75]: This should be `Gather and update all available and newest CVEs with their PoC.`

File: README.md [hunk 2]
```diff
@@ -86,6 +87,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Payloads All The Things](https://swisskyrepo.github.io/PayloadsAllTheThingsWeb/) - A list of useful payloads and bypasses for Web Application Security
 - [XSS Payloads](http://www.xss-payloads.com/) - The wonderland of JavaScript unexpected usages, and more
 - [exploitalert.com](https://www.exploitalert.com/search-results.html) - Database of Exploits
+- [Reverse Shell generator](https://www.revshells.com/) - Plug in attacker machine IP and port and generate any kind of reverse shell on the fly.
```
Comment (edoardottt) [inline README.md:90]: This should be`Online Reverse Shell generator with Local Storage functionality, URI & Base64 Encoding, MSFVenom Generator, and Raw Mode.`

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Comment by edoardottt** [comment]
you should push another commit(s) with the requested changes. when done, you should click on all the "resolve conversation" buttons. When done, I'll merge anything. I'm here for help :)

**Review by edoardottt — APPROVED** [review]

**Comment by edoardottt** [comment]
Amazing!! Thank you so much for your contribution @intrudir !

Hope you have a good day and more contribs ahead :)

**Pull request merged**